regression-tests.api/test_Cache.py

```diff
@@ -0,0 +1,22 @@
+from test_helper import ApiTestCase, is_auth, is_recursor
+
+
+class Servers(ApiTestCase):
```

Comment:
I don't see an actual test to see if the cache indeed has been flushed? Will this be done when cache-snooping is implemented in the API?